destinations/data/greptime.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: internal.odigos.io/v1beta1
+kind: Destination
+metadata:
+  type: greptime
+  displayName: GreptimeDB
+  category: managed
+spec:
+  image: greptime.svg
+  signals:
+    traces:
+      supported: false
+    metrics:
+      supported: true
+    logs:
+      supported: false
+  fields:
+    - name: GREPTIME_ENDPOINT
+      displayName: OTLP HTTP Endpoint
+      componentType: input
+      componentProps:
+        type: text
+        required: true
+        placeholder: https://<unique_id>.<aws_region>.aws.greptime.cloud
+        tooltip: Do not include the path `/v1/otlp/v1/metrics`.
```

Comment:
nice to have: in some other destinations, we check for this path and know to ignore it. so if the user enters it by mistake, things still work